common/app/common/dfp/DfpData.scala

```diff
@@ -457,7 +456,7 @@ case class GuCreativeTemplate(id: Long,
                               snippet: String,
                               creatives: Seq[GuCreative]) {
 
-  val examplePreviewUrl: Option[String] = creatives.headOption flatMap { creative => creative.previewUrl }
+  lazy val examplePreviewUrl: Option[String] = creatives flatMap {_.previewUrl} headOption
```

Comment:
Would be more efficient to find first creative with a preview URL, but this is more readable and given the small number of creatives this is better.